plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeWriter.java

```diff
@@ -169,8 +169,8 @@ public long getRowCount()
     public DataFileInfo getDataFileInfo()
             throws IOException
     {
-        List<String> dataColumnNames = columnHandles.stream().map(DeltaLakeColumnHandle::getName).collect(toImmutableList());
-        List<Type> dataColumnTypes = columnHandles.stream().map(DeltaLakeColumnHandle::getType).collect(toImmutableList());
+        List<String> dataColumnNames = columnHandles.stream().map(DeltaLakeColumnHandle::getPhysicalName).collect(toImmutableList());
```

Comment:
Do note that there are several types of column mapping in `io.trino.plugin.deltalake.transactionlog.DeltaLakeSchemaSupport.ColumnMappingMode` and we need to be able to handle them all.

Reply:
Sure. both physical name and type fields contain original name and type correspondingly when no column mapping defined. that allows to handle all modes.